### PR TITLE
Issue110-Unit test: Column string (case: Not Ends With) 

### PR DESCRIPTION
**Pull request by ArCiGoUnoSquare**
Unit test added for column filtering (string). Case: `Not Ends With`.

### CODE REVIEW

File: test/LocalDataSource.spec.tsx [hunk 1]
```diff
@@ -309,6 +310,19 @@ describe('LocalDataSource', () => {
         done();
       });
     });
+
+    // Not Ends With
```
Comment (geoperez) [inline test/LocalDataSource.spec.tsx]: Again the top comment?

File: test/LocalDataSource.spec.tsx [hunk 2]
```diff
@@ -309,6 +310,19 @@ describe('LocalDataSource', () => {
...
+    it('should return a payload with records where CustomerName not ends with \'a\'', (done) => {
+      dataSource.columns[1].Filter.Text = 'a';
+      dataSource.columns[1].Filter.Operator = 'NotEndsWith';
```
Comment (geoperez) [inline test/LocalDataSource.spec.tsx]: There is not an enum for Filter Operator?

File: test/LocalDataSource.spec.tsx (hunk 1 above)
Comment (ArCiGoUnoSquare) [reply test/LocalDataSource.spec.tsx]: Let me correct it

File: test/LocalDataSource.spec.tsx (hunk 2 above)
Comment (ArCiGoUnoSquare) [reply test/LocalDataSource.spec.tsx]: Updated, @geoperez

**Pull request merged**